app/controllers/questions_controller.rb

```diff
@@ -55,6 +55,10 @@ def search
     @questions = Question.search(params[:q])
   end
 
+  def by_tags
+    @questions = Question.by_tags(params[:tags_id])
```

Comment:
What do you think about reusing the `question/index` here, Since they are basically returning the same thing? And we will also get the `pagination` attributes for free when we pass this `@questions` through the `PaginationPresenter`. Just like we did in the `top_questions` action. 💭